packages/wonder-blocks-button/src/components/__tests__/activity-button.test.tsx

```diff
@@ -60,4 +61,48 @@ describe("ActivityButton", () => {
             expect(tab).toHaveAttribute("role");
         });
     });
+
+    describe("icons", () => {
+        test("icon is displayed with aria-hidden=true when the button contains a custom startIcon", async () => {
```

Comment:
We test for `aria-hidden=true` since this is applied to Phosphor icons in ActivityButton and Button components!